WireAnalytics/Sources/WireAnalytics/AnalyticsManagerProtocol.swift

```diff
@@ -28,9 +28,11 @@ public protocol AnalyticsManagerProtocol {
     func disableTracking()
 
     /// Enables analytics tracking and reporting.
-    /// This method should be used to re-enable analytics tracking after it has been disabled. Ensure that any necessary state or user context is correctly restored before resuming tracking.
+    /// This method should be used to re-enable analytics tracking after it has been disabled.
     /// - Parameter userProfile: The profile of the user to enable analytics for.
     /// - Returns: A new analytics session for the user.
+    ///
+    /// - Warning: Ensure that any necessary state or user context is correctly restored before resuming tracking.
```

Comment:
Is this warning relevant for the caller of the method?

Reply:
Probably not. I removed it for now. https://github.com/wireapp/wire-ios/commit/fc11fef203a482afe637e069c08381b8d64e956b